front/app/components/Form/Components/Controls/MultiSelectControl.tsx

```diff
@@ -60,6 +60,7 @@ const MultiSelectControl = ({
             }}
             inputId={sanitizeForClassname(id)}
             disabled={uischema?.options?.readonly}
+            isSearchable={false}
```

Comment:
Only did for the few citizen-facing forms using this specific version of multi-select because the search function may still be useful in the admin.